redi-connect-front/src/components/organisms/SideMenu.scss

```diff
@@ -16,15 +26,44 @@
       padding: 0.8rem 0;
       margin-right: -1rem;
 
+      @include mobile() {
+        display: inline-flex;
+        align-items: center;
+        flex-direction: column;
+        font-size: $size-6;
+        letter-spacing: 0;
```

Comment:
same here, don't overwrite yourself in case its not needed. set the properties for the case only

Reply:
done